assemblies/dynamic-plugins/assembly-front-end-plugin-wiring.adoc

```diff
@@ -0,0 +1,55 @@
+[id="assembly-front-end-plugin-wiring_{context}"]
+= Front-end plugin wiring
+:context: assembly-front-end-plugin-wiring
+
+You can configure front-end plugins to do the following:
+
+* Extend the internal library of available icons
+* Declare a new full page by defining a new route
+* Extend to an existing page using router bindings
+* Use mount points within the application
+* Provide additional utility APIs or replace existing ones
```

Comment:
I'm not sure is it worth adding mention of the SignInPage, scaffolder or techdocs extensions etc in this list?  This seems like a high-level overview of features, so maybe it doesn't need to be an exhaustive list I guess.